client/src/components/ui/table.tsx

```diff
@@ -1,77 +1,94 @@
-import {  useState } from 'react'
+import { useState } from 'react'
 import { Table, TableHeader, Column, Row, TableBody, Cell } from 'react-aria-components'
 
```

Comment:
This would be an appropriate place to declare the cellFormatter

```suggestion

  // Cell formatter function - only formats status column in sub-rows
  const statusCellFormatter = (
    cell: string, 
    _rowIndex: number, 
    cellIndex: number, 
    isSubRow: boolean
  ): CellFormat => {
    // Only format the last column (Status - index 6) in sub-rows
    if (isSubRow && cellIndex === 6) {
      const statusStyles: Record<string, string> = {
        'Granted': 'bg-accepted text-light rounded-md px-[16px] py-[4px]',
        'Expired': 'bg-expired text-light rounded-md px-[16px] py-[4px]',
        'Deffered': 'bg-deferred text-dark rounded-md px-[16px] py-[4px]',
      }
      
      return {
        content: cell,
        className: statusStyles[cell] || ''
      }
    }
    
    // Return cell unchanged for all other cases
    return { content: cell }
  }
  
```